samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -123,5 +123,43 @@ public async Task AssignmentsEvolvableEnumsAsync()
                 Console.WriteLine($"AssignmentsEvolvableEnumsAsync: {ex.ToString()}");
             }
         }
-    }
+
+        /// <summary>
+        /// Workflow to create and patch Assignment
```

Comment:
Workflow to create a new assignment, followed by an update to the assignment display name (aka title)

Reply:
updated to "Workflow to create a new assignment, followed by an update to the assignment display name (aka title)"